version/1/4/es/code_of_conduct.md

```diff
@@ -1,42 +1,42 @@
-# Código de Conducta convenido para Contribuyentes
+# Convenio de Código de Conducta para Contribuyentes
 
 ## Nuestro compromiso
 
-En el interés de fomentar una comunidad abierta y acogedora, nosotros como contribuyentes y administradores nos comprometemos a hacer de la participación en nuestro proyecto y nuestra comunidad una experiencia libre de acoso para todos, independientemente de la edad, dimensión corporal, discapacidad, etnia, identidad y expresión de género, nivel de experiencia, nacionalidad, apariencia física, raza, religión, identidad u orientación sexual. 
+Con el interés de fomentar una comunidad abierta y acogedora, nosotros como contribuyentes y administradores, nos comprometemos a hacer de la participación en nuestro proyecto y nuestra comunidad una experiencia libre de acoso para todos, independientemente de la edad, dimensión corporal, discapacidad, etnia, identidad y expresión de género, nivel de experiencia, nacionalidad, apariencia física, raza, religión, identidad u orientación sexual.
```

Comment:
I'd either not add the comma after administradores, or if adding it, then add a comma after nosotros:
, nosotros, como contribuyentes y administradores, ...

otherwise the reading rhythm feels weird....